setup.cfg

```diff
@@ -98,7 +98,12 @@ filterwarnings =
 
 [mypy]
 plugins = mypy_django_plugin.main
-disallow_untyped_calls = true
+show_error_codes = true
+strict = true
+warn_unreachable = true
+
+[mypy-django_countries.tests.*]
+allow_untyped_defs = True
```

Comment:
don't require every test function to be fully typed - this saves a lot of `-> None` definitions on test methods.